fonts.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+    font-family: 'ptserif';
+    src: url('ptserif-bold-webfont.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'ptserif';
+    src: url('ptserif-bolditalic-webfont.woff') format('woff');
+    font-weight: bold;
+    font-style: italic;
+}
+
+
+@font-face {
+    font-family: 'ptserif';
+    src: url('ptserif-italic-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)